e2e-tests/tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
```

Comment:
I'm not sure I understand the concerns about having these configuration files in the root, but it is not super important to me. It's mostly for convenience, and reduces the number of places where we deviate from defaults. As for moving the tests to another repository, we've already seen how that can harm visibility and developer ownership of these tests, so it is super important to me that they stay in the main repo.